Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1992,6 +1995,42 @@ def lhegensim2018ml(fragment,howMuch):
                             '-n':'10'
                             },steps['RECOHID15']])
 
+steps['RAWPRIMEHI18']={ '--scenario':'pp',
```

Comment:
I am wondering if we want to make this `RAW'` nomenclature to stick. Doesn't seem to be very future-proof (what about `RAW''`, etc.).

Reply:
RAW' to me just indicates that the data is partially processed at HLT, it's not about versioning.  The next iteration would not be RAW'', it would still be RAW'.   But ok, I'm completely agnostic about this terminology.  I'm happy to call it whatever you suggest ;-)